python-checks/src/test/resources/checks/verifiedSslTlsCertificateCheck.py

```diff
@@ -0,0 +1,282 @@
+  # requests-tests.py
+  # Main changes: the position of the error message in `kwargs`-cases has been moved closer to kwargs, not to
+  # the method invocation.
+  import requests
+
+  requests.request('GET', 'https://example.domain', verify=False) # Noncompliant {{Disabling certificate verification is dangerous.}}
+  #                                                        ^^^^^
+  requests.request('GET', 'https://example.domain', verify='') # Noncompliant {{Disabling certificate verification is dangerous.}}
+  #                                                        ^^
+  requests.request('GET', 'https://example.domain', verify=0) # Noncompliant {{Disabling certificate verification is dangerous.}}
+  #                                                        ^
+  requests.request('GET', 'https://example.domain', verify=0.0) # Noncompliant {{Disabling certificate verification is dangerous.}}
+  #                                                        ^^^
+  requests.request('GET', 'https://example.domain', verify=0j) # Noncompliant {{Disabling certificate verification is dangerous.}}
+  #                                                        ^^
+  requests.request('GET', 'https://example.domain', verify="") # Noncompliant {{Disabling certificate verification is dangerous.}}
+  #                                                        ^^
+  requests.request('GET', 'https://example.domain', verify=b'') # Noncompliant {{Disabling certificate verification is dangerous.}}
+  #                                                        ^^^
+  requests.request('GET', 'https://example.domain', verify=[]) # Noncompliant {{Disabling certificate verification is dangerous.}}
+  #                                                        ^^
+  requests.request('GET', 'https://example.domain', verify={}) # Noncompliant {{Disabling certificate verification is dangerous.}}
+  #                                                        ^^
+  requests.request('GET', 'https://example.domain', verify=()) # Noncompliant {{Disabling certificate verification is dangerous.}}
+  #                                                        ^^
+  requests.request('GET', 'https://example.domain', verify=set()) # Noncompliant {{Disabling certificate verification is dangerous.}}
+  #                                                        ^^^^^
+  requests.request('GET', 'https://example.domain', verify=range(0)) # Noncompliant {{Disabling certificate verification is dangerous.}}
+  #                                                        ^^^^^^^^
+  requests.request(verify=False, method='GET', url='https://example.domain') # Noncompliant {{Disabling certificate verification is dangerous.}}
+  #                       ^^^^^
+  kargs1 = {'verify': False} # Noncompliant {{Disabling certificate verification is dangerous.}}
+  #                   ^^^^^
+  requests.request('GET', 'https://example.domain', **kargs1)
```

Comment:
I believe the primary location of the issue should stay on `kargs1` argument for consistency. We can indeed add a secondary location to the assignment at line `101`

Reply:
I've added both the `verify: False` and `**kwargs` positions to the `PreciseIssue`. I've left the main issue message closer to the `verify: False` in all cases, so that it's hopefully easier to see the connection between the "verify" variable name and the "certificate verification" phrase in the issue description. Furthermore, one might say, that this is the position that is more likely to require a change by the user. I've additionally added the unpacking `**kwargs` as the secondary issue position, though. Now the connection to the `requests.request(...)` is easier to see, even though it's less likely to be modified by the user. I've also checked it in the UI: it now looks reasonably intuitive to me.

Reply:
ok makes sense. thanks for the clarification!